### DISCUSSION TITLE
Allow adjusting password reset & email change URL

### DISCUSSION THREAD

**Discussion by YugoCode** [pocketbase/pocketbase#1901] [Q&A] [answered]
The URLs sent via mail for a password reset and email change request look like this:
- `https://example.com/_/#/auth/confirm-password-reset/[token]`
- `https://example.com/_/#/auth/confirm-email-change/[token]`

Why does the `_/#/` part need to be there? Could we somehow remove it? I know that PocketBase can handle both requests out of the box, but I would like to add my own UI and do it myself. Now in SvelteKit I have to add folders to support the routes and I would like to be able construct the endpoint myself, instead of having to add routes for `_/#`. 

Is that somehow possible? Or is the preferred solution to create NGINX rules that forward e.g. `_/#/auth/confirm-password-reset/[token]` to my desired endpoint `/auth/confirm/password_reset`?

**Comment by ganigeorgiev** [comment, marked as answer]
You can change the default email confirmation urls by editing the email templates "Action URL" field in the "Admin UI > Settings > Mail settings":

![email_templates](https://i.imgur.com/95fQQMR.png)
